backend/server.py

```
@@ -366,75 +529,111 @@ def generate_gform():
+            }),
+            400,
+        )
+
+    try:
+        content = file_processor.process_file(uploaded_file)
+        if content:
+            logger.info("Processed uploaded file: %s", uploaded_file.filename)
+            return jsonify({"content": content})
+        else:
+            return jsonify({"error": "Could not extract content from the file"}), 400
+    except Exception as e:
```

Ruff (0.14.14)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)